src/test/java/LinkedList/LinkedListTest.java

```diff
@@ -59,39 +77,75 @@ public void testIncludes() {
     @Test
     public void testPrint() {
 
+//        LinkedList oneList  = new LinkedList();
```

Comment:
don't leave zombie code around! Make sure you delete stuff instead of leaving it all commented-out, for your own future sanity.